plugins/wpe-headless/.env.testing.example

```diff
@@ -0,0 +1,25 @@
+WP_ROOT_FOLDER=/var/www/html
```

Comment:
Is it worth git ignoring these paths in `plugins/wpe-headless/.gitignore` to prevent accidental commits?

```
.env.testing
/tests/_output/
/tests/_support/
```